Tests/UnitTestsSerialPort/SerialTests.cs

```diff
@@ -36,42 +38,75 @@ public void SetupComPorts()
                 Configuration.SetPinFunction(27, DeviceFunction.COM3_RTS);
                 Configuration.SetPinFunction(14, DeviceFunction.COM3_CTS);
 
-                Debug.WriteLine("You will need to connect:");
-                Debug.WriteLine("  COM2 RX  <-> COM3 TX");
-                Debug.WriteLine("  COM2 TX  <-> COM3 RX");
-                Debug.WriteLine("  COM2 RTS <-> COM3 CTS");
-                Debug.WriteLine("  COM2 CTS <-> COM3 RTS");
+                OutputHelper.WriteLine("You will need to connect:");
+                OutputHelper.WriteLine("  COM2 RX  <-> COM3 TX");
+                OutputHelper.WriteLine("  COM2 TX  <-> COM3 RX");
+                OutputHelper.WriteLine("  COM2 RTS <-> COM3 CTS"); // TODO: Optional?!
+                OutputHelper.WriteLine("  COM2 CTS <-> COM3 RTS"); // TODO: Optional?!
                 _serOne = new SerialPort("COM2");
                 _serTwo = new SerialPort("COM3");
-                Debug.WriteLine("Devices created, trying to open them");
+                OutputHelper.WriteLine("SerialPorts created, trying to open them");
                 _serOne.Open();
-                Debug.WriteLine("Serial One COM2 opened");
+                OutputHelper.WriteLine("SerialPort One COM2 opened.");
                 _serTwo.Open();
-                Debug.WriteLine("Devices opened, will close them");
-                // Wait a bit just to make sure and close them again
+                OutputHelper.WriteLine("SerialPort Two COM3 opened.");
+                OutputHelper.WriteLine("All SerialPorts opened, will close them");
+                // Wait a bit just to make sure and close them all
                 Thread.Sleep(100);
-                _serOne.Close();
-                _serTwo.Close();
+                EnsurePortsClosed();
+                OutputHelper.WriteLine("SerialPorts Closed.");
             }
             catch
             {
                 Assert.SkipTest("Serial Ports not supported in this platform or not properly configured");
             }
         }
 
+        //[Setup]
```

Comment:
just add an additional comment on top saying that this can be un commented to run on this board.

Reply:
Hopefully more helpful now.